celery_bungiesearch/signals.py

```diff
@@ -9,6 +9,16 @@ class CelerySignalProcessor(BungieSignalProcessor):
 
     ALLOWED_ACTIONS = ('save', 'delete')
 
+    def setup(self, model):
+        post_save.connect(self.enqueue_save, sender=model)
+        pre_delete.connect(self.enqueue_delete, sender=model)
+
+        self.update_task = get_update_task()
```

Comment:
minor nit: setup and teardown are not symmetric you should null update_task in teardown.